### ISSUE TITLE
does not work since mid-December 2018 ....

### ISSUE THREAD

**Issue by jbltpcblt** [gboudreau/nissan-connect-php#23]

**Comment by joeshaw** [comment]
Looks like the base URL has changed again.  See https://github.com/jdhorne/pycarwings2/issues/35

**Comment by nricklin** [comment]
The new URL works for me, however things aren't fully working.  When I make an async request, receive a resultkey, then try to use the resultkey on a subsequent request, the results never come.

Note sure if this is related to the API change, or perhaps something on my end.

**Comment by joeshaw** [comment]
Yes, it looks like there's a larger API change here.  Any request after the first one after logging in fails with a 401 error.  It looks like there might be extra state that needs to be sent along on each request.

Unfortunately I am in the US, which uses a totally separate API, so I cannot sniff the traffic.  If someone is in Europe and can use mitmproxy to capture a couple of flows from the official app, that'd be very helpful for fixing this.

**Comment by nricklin** [comment]
What's the totally separate API for the us?  I'm in the US too.  Currently using this URL to access the API (https://github.com/nricklin/leafpy/commit/9989a1531fe6620085d879d4e333361ad8aee095) with success, except for the above-mentioned async issue.

**Comment by joeshaw** [comment]
@nricklin Some links for background: https://github.com/joeshaw/carwings/issues/3

As far as I know, none of the popular carwings/nissan connect implementations implement the new API, because it only works in the US.  The other API works in the US and internationally.

**Comment by cedric222** [comment]
Unfortunately even the official nissan iphone apps don't work in Europe...  I cannot sniff the traffic !!!
( startClimateControl contine to work with nissan-connect-php and the official apps, but not the getStatus )

**Comment by gboudreau** [comment]
Try the latest version; I merged #24 which could resolve this issue.

**Comment by cedric222** [comment]
Yes, it's ok for france, with the mr #24
